packages/eslint-plugin-ecmascript-compat/lib/rule.js

```diff
@@ -4,16 +4,33 @@ const features = require('./features');
 const targetRuntimes = require('./targetRuntimes');
 
 const targets = targetRuntimes();
-const delegateeConfigs = compatibility
-  .forbiddenFeatures(features, targets)
-  .map((feature) => feature.ruleConfig);
+const unsupportedFeatures = compatibility.unsupportedFeatures(features, targets);
 
 module.exports = {
   meta: {
     type: 'problem',
-    schema: [], // no options
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          polyfills: {
+            type: 'array',
+            items: {
+              type: 'string',
+            },
+          },
+        },
+        additionalProperties: false,
+      },
+    ],
   },
   create(context) {
+    const polyfills = context.options?.[0]?.polyfills ?? [];
+    const forbiddenFeatures = unsupportedFeatures.filter(
+      (feature) => !polyfills.includes(feature.polyfill)
+    );
+
+    const delegateeConfigs = forbiddenFeatures.map((feature) => feature.ruleConfig);
```

Comment:
Let's do without this variable I had, and just do `feature.ruleConfig` when calling `createDelegatee()`